js_modules/dagster-ui/packages/ui-core/src/assets/useAssetCatalogFiltering.tsx

```diff
@@ -141,5 +141,7 @@ export function useAssetCatalogFiltering(
     filterInput,
     isFiltered,
     filtered,
+    computeKindFilter,
```

Comment:
I noticed line 129 isn't checking filters.storageKindTags?.length, can we add that